app/helpers/data_request_helper.rb

```diff
@@ -0,0 +1,6 @@
+module DataRequestHelper
+  def only_branston_registry_email
+    recipient_emails = instance_variable_get(:@recipient_emails)
```

Comment:
You don't need to use `instance_variable_get` here as the instance variable will be available here, although rubocop would complain about it's use.

The reasons for Rubocop's complaint can be seen here - https://www.rubydoc.info/gems/rubocop-rails/RuboCop/Cop/Rails/HelperInstanceVariable.

There is usually a good reason for the rubocop warning but sometimes it might be right to ignore it and you can disable a rule by adding a comment on the offending line e.g:
```
some line of code with an @instance_var # rubocop:disable Rails::HelperInstanceVariable
```
or you can disable it for multiple lines:
```
# rubocop:disable Rails::HelperInstanceVariable
line with @instance_var
line with another @var
and another @var
# rubocop:enable Rails:HelperInstanceVariable
```

Reply:
Thanks for the tip, I did get the rubocop error so I used that variable. I will amend